examples/in_toto_run_test.rs

```diff
@@ -19,5 +19,5 @@ fn main() {
     .unwrap();
     let json = serde_json::to_value(&link).unwrap();
 
-    println!("Generated link: {}", json)
+    println!("{json:#}")
```

Comment:
Since this binary has a hard-coded `run` command, it would not be used for piping or shell scripts; only for show.

I believe this is an example meant to show how this works.

Reply:
```suggestion
    println!("Generated link: {json:#}")
```

Reply:
I like using `jq` to explore the output. If it's being formatted, I guess it doesn't matter much.